certified-connectors/SigmaConsoCR/apiDefinition.swagger.json

```diff
@@ -0,0 +1,292 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "title": "Sigma Conso CR",
+      "description": "Sigma Conso Consolidation & Reporting Connector provides a set of APIs to interact with your Consolidation & Reporting instances. Easily transfer data, trigger actions and automate workflows in Sigma Conso. Go beyond the limits of traditional CPM solutions.",
+      "version": "1.0",
+      "contact": {
+        "name": "Sigma Conso",
+        "url": "https://www.sigmaconso.com/",
+        "email": "support@sigmaconso.com"
+      }
+    },
+    "host": "x",
+    "basePath": "/",
+    "schemes": [
+      "https"
+    ],
+    "consumes": [],
+    "produces": [],
+    "paths": {
+      "/api/job/process": {
+        "post": {
+          "responses": {
+            "200": {
+              "description": "default",
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "Message": {
+                    "type": "string",
+                    "description": "Message"
+                  }
+                }
+              }
+            }
+          },
+          "summary": "Run a process",
+          "description": "Run a process",
+          "operationId": "Processjob",
+          "parameters": [
+            {
+              "name": "ApplicationURL",
+              "in": "query",
+              "required": true,
+              "type": "string",
+              "description": "Enter the application url.",
+              "x-ms-summary": "Enter the application url."
+            },
+            {
+              "name": "body",
+              "in": "body",
+              "required": true,
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "Process": {
+                    "type": "string",
+                    "description": "Process"
+                  },
+                  "Action": {
+                    "type": "string",
+                    "description": "Action",
+                    "title": ""
+                  },
+                  "CustomerCode": {
+                    "type": "string",
+                    "description": "CustomerCode",
```

Comment:
@tve-sigmaconso The description values should be free from grammatical errors, Can it be `"Customer Code"` ?

Reply:
This is fixed in commit 47bbf27e